backend/reviews/handler.js

```diff
@@ -117,10 +117,11 @@ async function scrape(appId, scraper, store) {
     }
     // no need to throttle Apple App Store
     else{
-      promises.push(scraper.reviews({
```

Comment:
Why do we need to save this to a variable first? (Not disputing this change, just wondering what this code does :) )